packages/assets-controllers/src/NftController.ts

```diff
@@ -52,6 +56,77 @@ type SuggestedNftMeta = {
   origin: string;
 };
 
+export enum OpenSeaV2ChainIds {
+  ethereum = 'ethereum',
+}
+
+export type OpenSeaV2GetNftResponse = { nft: OpenSeaV2DetailedNft };
+
+export type OpenSeaV2Nft = {
+  identifier: string;
+  collection: string;
+  contract: string;
+  token_standard: string;
+  name: string;
+  description: string;
+  image_url: string;
+  metadata_url: string;
+  updated_at: string;
+  is_disabled: boolean;
+  is_nsfw: boolean;
+};
+
+export type OpenSeaV2DetailedNft = OpenSeaV2Nft & {
+  animation_url: string;
+  is_suspicious: boolean;
+  creator: string;
+  traits: {
+    trait_type: string;
+    display_type: string;
+    max_value: string;
+    trait_count: number;
+    value: number | string;
+  }[];
```

Comment:
More optional fields

```suggestion
  animation_url?: string;
  is_suspicious: boolean;
  creator: string;
  traits: {
    trait_type: string;
    display_type?: string;
    max_value: string;
    trait_count?: number;
    value: number | string;
  }[];
```